common/vm_set_power_state.yml

```diff
@@ -69,13 +73,13 @@
 - name: "Change VM power state failure"
   ansible.builtin.fail:
     msg: >
-      {% if vm_change_power_state is defined and vm_change_power_state.msg is defined %}{{ vm_change_power_state.msg }}
-      {% else %}Failed to set VM power state to {{ vm_power_state_set | lower }}{% endif %}
+      Failed to set VM power state to {{ vm_power_state_set | lower }} because of:
+      {{ vm_change_power_state.msg | default('') }}
   when: >
```

Comment:
can also put the when condition to the beginning of the block

Reply:
Updated.